source/MRViewer/MRShortcutManager.h

```diff
@@ -52,7 +76,9 @@ class MRVIEWER_CLASS ShortcutManager
     MRVIEWER_API virtual bool processShortcut( const ShortcutKey& key, Reason = Reason::KeyDown ) const;
 
     // make string form key and returns it
-    MRVIEWER_API static std::string getKeyString( const ShortcutKey& key );
+    MRVIEWER_API static std::string getKeyString( const ShortcutKey& key, bool respectLastKey = true );
+    // calculates paddings needed for drawing shortcut key with modifiers
+    MRVIEWER_API static float getKeyPaddings( const ShortcutKey& key, float scaling );
```

Comment:
I am not sure that paddings should be here, paddings is visual option, and this class should not be responsible for visualization